kite-data/kite-data-hbase/src/main/java/org/kitesdk/data/hbase/impl/SchemaManager.java

```diff
@@ -196,4 +196,55 @@ public void migrateSchema(String tableName, String entityName,
    * @return The table names with managed schemas.
    */
   public Collection<String> getTableNames();
+
+  /**
+   * Sets the properties on the Managed Schema.
+   * All existing properties are removed.
+   *
+   * @param tableName  The table name of the managed schema
+   * @param entityName The entity name of the managed schema
+   * @param properties Map<String, String> properties
+   */
+  void setProperties(String tableName, String entityName,
```

Comment:
What about `getProperties(String table, String entity)` that returns `Map<String, String>`? Why not include it?